docs/README.md

```diff
@@ -1 +1,116 @@
-# Checkout Components
+📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# CHECKOUT COMPONENTS
+
+⚠️⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation, use it at your own risk.** ⚠️⚠️
+
+<!-- DOCS-IGNORE:start -->
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- DOCS-IGNORE:end -->
+
+Under the app's name, you should explain the topic, giving a **brief description** of its **functionality** in a store when installed.
+
+Next, **add media** (either an image of a GIF) with the rendered components, so that users can better understand how the app works in practice. 
+
+![Media Placeholder](https://user-images.githubusercontent.com/52087100/71204177-42ca4f80-227e-11ea-89e6-e92e65370c69.png)
```

Comment:
This is an instruction to clarify which info should be added to the doc - It should be replaced with real data!